README.md

```diff
@@ -173,6 +173,30 @@ are detailed below:
   If you are an author of an original template not affiliated with any
   organization, only the standard package naming guidelines apply to you.
 
+- **Description:** Write simple, easily-understandable and succinct
+	descriptions.
+	- Keep it short. Try to maximise the content:length ratio and weigh your words
+		thoughtfully. No more than 100 characters, preferably much less.
+	- Avoid the word ‘Typst’, which is redundant unless your package or template
+		actually has to do with Typst itself or its ecosystem (like in the case of
+		[Mantys](https://typst.app/universe/package/mantys/) or
+		[t4t](https://typst.app/universe/package/t4t)).
+	- Don’t terminate your description with a full stop.
```

Comment:
Can you swap this point and the previous one, so that the two "Avoid…" points are next to each other?